src/style.css

```diff
@@ -1,3 +1,70 @@
+:root {
+  --color: rgba(0, 172, 220, 1);
+}
+
+html {
+  font-weight: 500;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  justify-content: space-between;
+  text-wrap: nowrap;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+a {
+  display: flex;
+  align-content: center;
+  align-items: center;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav_item {
+  color: transparent;
+}
+
+.nav_link {
+  line-height: 14.06px;
+  height: 60px;
+  margin-left: 20px;
+  color: black;
+  text-decoration: none;
+  position: relative;
+  align-items: center;
+}
+
+.nav_link:hover {
+  color: var(--color);
+}
+
+.is-active {
+  margin-left: 0;
```

Comment:
```suggestion

```
This class should add just a color and the underline. Also, it should work properly for every link in the navbar. If you add a margin reset, it breaks the layout when applied to other navigation links. Consider adding a different rule to reset the margin for the first link